syntax = "proto3";

option csharp_namespace = "ClientApp.Common.NetMsgDefine";

enum MsgType
{
  // 未定义消息
  UNDEFINED = 0000;

  // 注册
  REGISTER_C2S = 1010;
  REGISTER_S2C = 2010;

  // 登录
  LOGIN_C2S = 1020;
  LOGIN_S2C = 2020;
};

message Msg_Register_C2S
{
  string Name = 1;
  string Password = 2;
}
message Msg_Register_S2C
{
  int32 result = 1;
}
message Msg_Login_C2S
{
  string Name = 1;
  string Password = 2;
}
message Msg_Login_S2C
{
  int32 result = 1;
  int32 id = 2;
}